docs/release-notes/0.10.8.md

```diff
@@ -5,7 +5,7 @@
 
 * Write out `64bit` indptr when appropriate for {func}`~anndata.experimental.concat_on_disk` {pr}`1493` {user}`ilan-gold`
 * Support for Numpy 2 {pr}`1499` {user}`flying-sheep`
-
+* `xfail` `~anndata.experimental.sparse_dataset` docstring test on account of new `~scipy` version {pr}`1514` {user}`ilan-gold`
```

Comment:
```suggestion
* {func}`~pytest.xfail` {func}`~anndata.experimental.sparse_dataset` docstring test on account of new {mod}`scipy` version {pr}`1514` {user}`ilan-gold`
```

you need to add roles here to make links. `~` is there to remove leading elements so, ``{func}`~foo.bar` `` becomes a link with the text `bar` while ``{mod}`~scipy` `` is identical to ``{mod}`scipy` ``